zboxcore/sdk/deleteworker.go

```diff
@@ -157,7 +176,7 @@ func (req *DeleteRequest) ProcessDelete() error {
 		if commitReq.result != nil {
 			if commitReq.result.Success {
 				Logger.Info("Commit success", commitReq.blobber.Baseurl)
-				req.consensus++
+				req.consensus.consensus++
```

Comment:
Update consensus with lock

Reply:
it is updated one by one in `for`. so lock is not required here.

Reply:
If the for loop is the only place multiple goroutines could happen to the req.consensus, then we only to define a local mutex to protect the consensus object and the `objectTreeRefs`, otherwise, if there are other places that will access the consensus data concurrently, then all the read/write actions must be protected by the lock. So if the `req.consensus.consensus++` is a safe action above, I suggest to remove the req.Mutex, and do it locally in the `ProcessDelete()` method.

Reply:
fixed